packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -601,9 +599,7 @@ export default function CustomDataForm(props) {
         <Button
           children=\\"empty\\"
           type=\\"reset\\"
-          onClick={() => {
-            resetStateValues();
-          }}
+          onClick={() => resetStateValues()}
```

Comment:
If you want to simplify even more, you can just set to onClick={resetStateValues}

Reply:
ah yes - updating in the next rev